src/SFA.DAS.ApplyService.Web/Controllers/ApplicationController.cs

```diff
@@ -58,48 +58,44 @@ public async Task<IActionResult> Applications()
 
             if (!applications.Any())
             {
+                //ON-2068 Registered org  with no application created via digital service then
+                //display empty list of application screen
                 if (org != null && org.RoEPAOApproved)
-                {
-                      return await StartApplication(userId);
-                }
+                    return View(applications);
 
-                if (org == null)
-                {
-                    if (await _userService.AssociateOrgFromClaimWithUser())
-                        return await StartApplication(userId);
+                if (org != null)
+                    return View("~/Views/Application/Declaration.cshtml");
 
-                    return RedirectToAction("Index", "OrganisationSearch");
-                }
+                if (await _userService.AssociateOrgFromClaimWithUser())
```

Comment:
Has the logic here changed? If it hasn't it's not necessary to change the code?

Reply:
Tided up the code the logic is the same